docs/en/user-guide.md

```diff
@@ -0,0 +1,42 @@
+# Dumpling manual
+
+[Dumpling](https://github.com/pingcap/dumpling) is a tool that supports export MySQL/TiDB data as SQL text or CSV file.
+
+It's designed to replace [Mydumper](https://github.com/pingcap/mydumper), so you can learn its basic usage from Mydumper. Of course, we didn't make a replica of Mydumper, so there are difference between them.
+
+The following table lists main parameters of dumpling.
```

Comment:
Better just copy the output from `./dumpling -h`.

Reply:
Sounds good, do we need to change Chinese document at the same time?

Reply:
Some of docs are copied from CLI output, and the description at CLI help message seems isn't clear as this doc, maybe leave this doc as is would also be good.

Reply:
> Sounds good, do we need to change Chinese document at the same time?

Well the `-h` output is English only 😐. You could change the Chinese doc to fit the meaning English but I think not much needs to be changed.

Reply:
Some of flags that `-h` prints sees isn't present at this doc(e.g. `--csv-null-value`), and this doc has mentioned `-h` flag at last.

I think it's better to keep consistency with Chinese doc version... Making a replica of existing resources seems a little strange... 🤔

Reply:
The documentations should be consistent with the actual behavior of the program, not with each other 🤔.